cmd/instances.go

```diff
@@ -838,6 +839,79 @@ var showSwiftPrefixInstanceCmd = &cobra.Command{
 	},
 }
 
+var appsVersionsCmd = &cobra.Command{
+	Use:     "apps-versions",
```

Comment:
I'd prefer to have this command in `cozy-stack apps versions` instead of `cozy-stack instances apps-versions`